barretenberg/cpp/src/barretenberg/aztec_ivc/aztec_ivc.test.cpp

```diff
@@ -32,8 +32,10 @@ class AztecIVCTests : public ::testing::Test {
 
     /**
      * @brief Construct mock circuit with arithmetic gates and goblin ops
-     * @details Currently default sized to 2^16 to match kernel. (Note: dummy op gates added to avoid non-zero
-     * polynomials will bump size to next power of 2)
+     * @details Defaulted to add 2^16 gates (which will bump to next power of two with the addition of dummy gates).
+     * The size of the baseline circuit needs to be ~2x the number of gates appended to the kernel circuits via
+     * recursive verifications (currently ~60k) to ensure that the circuits being folded are equal in size. (This is
+     * only necessary if the structured trace is not in use).
```

Comment:
I looked into making this more robust but its already as good as it can be without making the logic much more complicated. The circuits created here are just over a power-of-two limit so the amount we can add to them is maximized. If recursive verification gets more expensive then this might go over the limit again but that's just the way it is.